core/src/test/scala/org/apache/spark/scheduler/OutputCommitCoordinatorSuite.scala

```diff
@@ -270,6 +263,16 @@ class OutputCommitCoordinatorSuite extends SparkFunSuite with BeforeAndAfter {
       .stageStart(meq(retriedStage.head), any())
     verify(sc.env.outputCommitCoordinator).stageEnd(meq(retriedStage.head))
   }
+
+  test("SPARK-39195: Spark OutputCommitCoordinator should abort stage " +
```

Comment:
is this test the same as https://github.com/apache/spark/pull/36564/files#diff-d5d8811dcf8af13683cb2bac4a583577cb2accca0cd64182cf214bbd174b017cR47 ?

Reply:
Removed